addons/im_livechat/static/src/legacy/widgets/public_livechat_view/public_livechat_view.js

```diff
@@ -338,7 +338,7 @@ const PublicLivechatView = Widget.extend({
      _updateTimestamps() {
         const isAtBottom = this.isAtBottom();
         this.$('.o_mail_timestamp').each(function () {
-            const date = $(this).data('date');
+            const date = new Date($(this).data('date'));
```

Comment:
here the date format is I think "time.getLangDatetimeFormat" which uses the database format (for the current user), so I think we need to use this format to parse it (and not "new Date" that is rather limited) => so I think we need to use probably `parseDateTime`.